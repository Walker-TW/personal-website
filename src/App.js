import React from "react";
import logo from "./logo.svg";
import "./App.css";

function App() {
  return (
    <div className="App">
      <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>Let's Hire Tom Walker He's A Swell Guy!</p>
        <a
          className="App-link"
          href="https://github.com/Walker-TW"
          target="_blank"
          rel="noopener noreferrer"
        >
          Check Out My Projects
        </a>
      </header>
    </div>
  );
}

export default App;
