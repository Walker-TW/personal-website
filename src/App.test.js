import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

import { shallowToJson } from "enzyme-to-json";

import { shallow } from "enzyme";

describe("<App/>", () => {
  it("renders", () => {
    const wrapper = shallow(<App />);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});

test("renders github project link", () => {
  const { getByText } = render(<App />);
  const linkElement = getByText(/Check Out My Projects/i);
  expect(linkElement).toBeInTheDocument();
});
