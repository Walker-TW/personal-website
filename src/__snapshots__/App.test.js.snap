// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App/> renders 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Let's Hire Tom Walker He's A Swell Guy!
    </p>
    <a
      className="App-link"
      href="https://github.com/Walker-TW"
      rel="noopener noreferrer"
      target="_blank"
    >
      Check Out My Projects
    </a>
  </header>
</div>
`;
